code/game/objects/items/theft_items.dm

```diff
@@ -82,9 +120,20 @@
 	else
 		load(core, user)
 
+/obj/item/nuke_core_container/proc/crack_open()
+	visible_message("<span class='notice'>[src] bursts open!</span>")
```

Comment:
`warning` or `boldnotice` here instead, maybe? It might be easy to miss the message otherwise.

Reply:
Done!